### ISSUE TITLE
 returned non-zero exit status -9

### ISSUE THREAD

**Issue by rghelichi** [tensorflow/tensorflow#18984]
I am using gcloud ml-engine. I can run tensorflow package locally and on the cloud with non-distributed version. When I run in the distributed using STANDARD_1 it gives me the following error:
{
 insertId:  "tfdl1gg1hisuso"  
 jsonPayload: {
  created:  1525101438.307529   
  levelname:  "ERROR"   
  lineno:  889   
  message:  "Command '['python', '-m', u'trainer_1_4.task', u'--train-data-file', u'gs://main-175118-accern/tf_data/train_1*', u'--num-epochs', u'1', u'--train-batch-size', u'10', u'--eval-batch-size', u'10', u'--eval-data-file', u'gs://main-175118-accern/tf_data/test_1*', '--job-dir', u'gs://main-175118-accern/accern_1_4']' returned non-zero exit status -9"   
  pathname:  "/runcloudml.py"   
 }
 labels: {
  compute.googleapis.com/resource_id:  "1659456639612915557"   
  compute.googleapis.com/resource_name:  "cmle-training-worker-7bc72ccf46-0-qz5zj"   
  compute.googleapis.com/zone:  "us-central1-f"   
  ml.googleapis.com/job_id:  "accern_1_4_dist_1"   
  ml.googleapis.com/job_id/log_area:  "root"   
  ml.googleapis.com/task_name:  "worker-replica-0"   
  ml.googleapis.com/trial_id:  ""   
 }
 logName:  "projects/main-175118/logs/worker-replica-0"  
 receiveTimestamp:  "2018-04-30T15:17:23.426890073Z"  
 resource: {
  labels: {…}   
  type:  "ml_job"   
 }
 severity:  "ERROR"  
 timestamp:  "2018-04-30T15:17:18.307528972Z"  
}

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by rghelichi** [comment]
Have I written custom code: YES
OS Platform and Distribution: LINUX 16.04
TensorFlow installed from: Source
TensorFlow version: 1.5
Bazel version: 0.10.1
CUDA/cuDNN version: RUN ON THE CLOUD
GPU model and memory: RUN ON THE CLOUD
Exact command to reproduce: gcloud ml-engine jobs submit training

**Comment by jart** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!
